src/activate/activate.c

```diff
@@ -418,17 +413,15 @@ static int parse_argv(int argc, char *argv[]) {
                                 return log_oom();
 
                         STRV_FOREACH(s, names)
-                                if (!fdname_is_valid(*s)) {
-                                        _cleanup_free_ char *esc;
+                        if (!fdname_is_valid(*s)) {
```

Comment:
No no... Please do not re-indent this. `STRV_FOREACH()` is a loop.

Reply:
ok, i can change this in the settings...